services/gateway-api/src/circuit_breaker/anthropic_breaker.py

```
@@ -0,0 +1,360 @@
+
+class AnthropicCircuitBreaker:
+    """
+    Circuit breaker specifically for Anthropic Claude API
+
+    Features:
+    - Automatic failover to GPT-4 or Gemini
+    - Request caching
+    - Cost tracking
+    """
+
+    def __init__(
```

Ruff (0.14.8)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)